statistics/table.go

```diff
@@ -642,7 +642,7 @@ func (coll *HistColl) getEqualCondSelectivity(sc *stmtctx.StatementContext, idx
 	if minRowCount < idxCount {
 		return crossValidationSelectivity, nil
 	}
-	return idxCount / float64(idx.TotalRowCount()), nil
+	return float64(idx.QueryBytes(bytes)) / idx.TotalRowCount(), nil
```

Comment:
Seems like this line of change overwrites #20264's change. And line 641 above introduced by #20264 should be changed to `idx.QueryBytes`.

Reply:
Good catch. It is fixed now.